### ISSUE TITLE
24v powered board thermal swing

### ISSUE THREAD

**Issue by meteozond** [makerbase-mks/MKS-Robin-Nano-V1.X#102]
[There is some information](https://www.thingiverse.com/thing:3672543) about noisy 5v line if the board is powered via 24v  adapter. 

I've ran into this problem too on my v1.2. 

Temperature sensor is fluctuating up to 4-5 Celsius even with powered off heaters. 

![image](https://user-images.githubusercontent.com/975689/113047668-44393600-91aa-11eb-94f2-567438b9ffe3.png)

Powering over 5v usb or 12v power adapter eliminates this.

So what is the best way to fix it without replacing capacitors or power adapter to 12v?

**Comment by mks-viva** [comment]
Hi! You only connected temperature sensor on the board?

**Comment by meteozond** [comment]
> Hi! You only connected temperature sensor on the board?

Yes, after facing this problem I've removed all connected devices. Problem hasn't gone :( Connecting through usb only gives better picture:

![image](https://user-images.githubusercontent.com/975689/115038552-515b6200-9ed8-11eb-8eb0-fc66a9bdefb4.png)

I've tested two new power adapters 24v and 12v - both shows same picture:

24v
![image](https://user-images.githubusercontent.com/975689/115039127-d777a880-9ed8-11eb-86df-4864965883d5.png)

12v
![image](https://user-images.githubusercontent.com/975689/115039141-dc3c5c80-9ed8-11eb-92d4-de9b61c7aa46.png)

**Comment by meteozond** [comment]
Just received and tested several replacement cards:

- MKS Robin nano v1.2 - temperature jumps
- MKS Robin nano v3.0 - temperature jumps (wtf!!!)
- MKS SGEN L V2.0 - stable temperature reports

Looks like all Robin series boards give unstable temperature reporting what leads inconsistent prints 😡
